source/android/adaptivecards/src/androidTest/java/io/adaptivecards/objectmodel/TextBlockPropertiesTest.java

```diff
@@ -307,6 +309,19 @@ public void WrapTest() throws Exception
         }
     }
 
+    @Test
+    public void TestLineBreaks () throws Exception
```

Comment:
Should probably add a test with numbered lists as well since that's what we claim to actually support today with the '\r' escape character. [https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/text-features](https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/text-features)